packages/valory/skills/abstract_round_abci/models.py

```diff
@@ -305,44 +301,47 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
             "share_tm_config_on_startup", kwargs, bool
         )
         self.tendermint_p2p_url: str = self._ensure("tendermint_p2p_url", kwargs, str)
-        setup_params_: Dict[str, Any] = self._ensure("setup", kwargs, dict)
-
-        def check_val(val: Any, skill_id: str) -> List[Any]:
-            """Check that a value a list"""
-            if not isinstance(val, list):
-                raise TypeError(
-                    f"Value `{val}` in `setup` set in `models.params.args` of `skill.yaml` of `{skill_id}` is not a list"
-                )
-            return val
+        self.setup_params: Dict[str, Any] = self._ensure("setup", kwargs, dict)
 
         # we sanitize for null values as these are just kept for schema definitions
         skill_id = kwargs["skill_context"].skill_id
-        setup_params: Dict[str, List[Any]] = {
-            key: check_val(val, skill_id)
-            for key, val in setup_params_.items()
-            if val is not None
-        }
-        self.setup_params = setup_params
```

Comment:
Why this change?

Reply:
Added type check for the setup params below, so this check was duplicate and I removed it.